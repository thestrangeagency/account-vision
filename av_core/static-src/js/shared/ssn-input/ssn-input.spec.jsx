import React from 'react';
import { shallow } from 'enzyme';

import SsnInput from './ssn-input';

test('it renders', () => {
  const wrapper = shallow((
    <SsnInput
      id="date"
      name="date"
      placeholder="enter ssn here"
      required
    />
  ));

  expect(wrapper).toMatchSnapshot();
});
