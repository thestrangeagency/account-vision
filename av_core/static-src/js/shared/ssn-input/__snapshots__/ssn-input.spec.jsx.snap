// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<input
  className="form-control"
  defaultValue={null}
  id="date"
  name="date"
  pattern="^\\\\d{9}$"
  placeholder="enter ssn here"
  required={true}
  type="tel"
/>
`;
